tests/integration/validate/test_validate_command.py

```diff
@@ -170,7 +170,6 @@ def test_lint_supported_runtimes(self):
         supported_runtimes = [
             "dotnet8",
             "dotnet6",
-            "go1.x",
```

Comment:
I was just looking at the canaries, I think provided also needs to be removed right?

Reply:
Right it is confusing since all printed in the same line. Removed it as well.